src/test/java/uk/gov/hmcts/reform/draftstore/service/draftservice/BaseTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.draftstore.service.draftservice;
+
+import org.junit.Before;
+import org.mockito.BDDMockito;
+import org.mockito.Mock;
+import uk.gov.hmcts.reform.draftstore.data.DraftStoreDAO;
+import uk.gov.hmcts.reform.draftstore.domain.Draft;
+import uk.gov.hmcts.reform.draftstore.service.DraftService;
+import uk.gov.hmcts.reform.draftstore.service.UserAndService;
+
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+import static org.mockito.Matchers.anyInt;
+
+public class BaseTest {
```

Comment:
We could rename it to indicate that it's only base test for a certain type of test classes (it contains draft service and draft repo).

How about `BaseControllerTest`?

Reply:
notice it's in its own package

Reply:
right 👍